addons/mail/static/src/core/common/composer.js

```diff
@@ -576,7 +576,7 @@ export class Composer extends Component {
             default_partner_ids:
                 this.props.type === "note"
                     ? []
-                    : allRecipients.map((recipient) => recipient.persona.id),
+                    : allRecipients.map((recipient) => recipient.partner_id),
```

Comment:
I believe we still need to set ids and not records in the context here

Reply:
This doesn't work, for some reason, the partner_id is not a record but just the id.